packages/front/src/pages/user-profile.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+  <div class="user-profile">
+    <div class="user-profile__sidebar">
+      <div class="user-profile__sidebar-logo-wrapper">
+        <a href="/">
+          <img
+            src="../assets/square-logo-80.png"
+            alt="brand-logo"
+          >
+        </a>
+      </div>
+    </div>
+
+    <BModal
+      color="gray-30"
+      :open="deleteAccountModal"
+    >
+      <p>Are you sure you want to delete your account?</p>
+      <div class="user-profile__form-buttons">
+        <BButton
+          size="medium"
+          variant="transparent"
+          value="Cancel"
+          @click="hideDeleteAccountModal"
+        >
+          Cancel
+        </BButton>
+        <BButton
+          size="medium"
+          value="Delete"
+          @click="onDelete"
+        >
+          Delete
+        </BButton>
+      </div>
+    </BModal>
```

Comment:
eu percebi que aqui vocês tiveram que basicamente replicar o `FConfirmation` só que manualmente, daí percebi que ele era muito específico também, então [abri essa pr aqui](https://github.com/AsyncPlanningPoker/BrieflyPlanningPoker/pull/141) pra melhorar a usabilidade dele e a gente não precisar replicar código

daí podemos trocar esse cara pra:

```suggestion
    <BModal
      color="gray-30"
      :open="deleteAccountModal"
    >
      <FConfirmation
        action="delete"
        message="Are you sure you want to delete your account?"
        @close="hideDeleteAccountModal"
        @confirm="onDelete"
      />
    </BModal>
```

ah, tem q importar o `FConfirmation` se não o modal fica sem nada dentro kk